### ISSUE TITLE
Lab/Experiment Hosting Request for  Control Engineering lab

### ISSUE THREAD

**Issue by Githubdei** [virtual-labs/engineers-forum#828]
## *Lab/Experiment(s) Hosting Request*
Use this issue to get your lab and experiment repositories hosted on Virtual Labs. This issue also is used to host labs and their experiments that have been migrated to Phase 3 lab/exp format from Phase 2. 

Please provide item 1 for all the experiments of the lab

1. **Hosting Unit**: 
| No. 1|  Analyze the transfer function and pole-zero plot of a control system | [Experiment repository URL](https://github.com/virtual-labs/exp-pole-zero-plot-dei) | main/v1.0.0 |
2. **GitHub Handle**: Githubdei
3. **Virtual Lab Phase**: 3
4. **Existing Hosted URL**:
5. **Approved Proposal**:
[Proposal_Control.Engineering.Virtual.Lab_DEI_Dr.Rajeev.Kumar.Chauhan.pdf](https://github.com/virtual-labs/engineers-forum/files/9662736/Proposal_Control.Engineering.Virtual.Lab_DEI_Dr.Rajeev.Kumar.Chauhan.pdf)

**Comment by Githubdei** [comment]
We are ready with the first three experiments, so please move forward with the lab hosting process, links for all three experiments are provided in the issue above.

**Comment by Githubdei** [comment]
## *Lab/Experiment(s) Hosting Request*
Use this issue to get your lab and experiment repositories hosted on Virtual Labs. This issue also is used to host labs and their experiments that have been migrated to Phase 3 lab/exp format from Phase 2. 

Please provide item 1 for all the experiments of the lab

1. **Hosting Unit**: 
| No. 1|  Analyze the transfer function and pole-zero plot of a control system | [Experiment repository URL](https://github.com/virtual-labs/exp-pole-zero-plot-dei) | main/v1.0.1 |
| No. 2|   Find and Plot the step and impulse response for a first order unity feedback system | [Experiment repository URL](https://github.com/virtual-labs/exp-first-order-unity-feedback-system-dei) | main/v1.0.1 |
| No. 3|   Find and Plot the step and impulse response for a second order unity feedback system | [Experiment repository URL](https://github.com/virtual-labs/exp-second-order-unity-feedback-system-dei) | main/v1.0.1 |
| No. 4|  To determine impulse and step response of type '0', '1' and '2' system | [Experiment repository URL](https://github.com/virtual-labs/exp-type-0-1-2-systems-dei) | main/v1.0.0 |
2. **GitHub Handle**: Githubdei
3. **Virtual Lab Phase**: 3
4. **Existing Hosted URL**:
5. **Approved Proposal**:
[Proposal_Control.Engineering.Virtual.Lab_DEI_Dr.Rajeev.Kumar.Chauhan.pdf](https://github.com/virtual-labs/engineers-forum/files/9662736/Proposal_Control.Engineering.Virtual.Lab_DEI_Dr.Rajeev.Kumar.Chauhan.pdf)

**Comment by priya100raman** [comment]
@Githubdei, Could you kindly clarify if these experiments are towards meeting Extended Phase 3 targets? If so they need to go through the review process before we host them on production. However, if they are Phase 2 experiments being converted into Phase 3 UI and not towards meeting Extended Phase 3 targets we can host them. Please clarify

**Comment by Githubdei** [comment]
Yes, these experiments are towards meeting extended phase 3. The review process has been completed. Therefore, we have raised the request to host these 4 experiments of control Engineering. Kindly do needful. I shall be thankful to you for this support.

**Comment by vlabsdev-iitd** [comment]
@Githubdei @priya100raman

Before hosting of any experiment/lab, the final review by external expert committee should be carried out and then only we can host them. Please wait until the final review happens.
